packages/event-handler/tests/unit/rest/Router/basic-routing.test.ts

```diff
@@ -144,4 +144,47 @@ describe('Class: Router - Basic Routing', () => {
     expect(JSON.parse(createResult.body).actualPath).toBe('/todos');
     expect(JSON.parse(getResult.body).actualPath).toBe('/todos/1');
   });
+
+  it('routes to the included router when using split routers', async () => {
+    // Prepare
+    const baseRouter = new Router();
+    baseRouter.get('/', async () => ({ api: 'root' }));
+    baseRouter.get('/version', async () => ({ api: 'listVersions' }));
+    baseRouter.get('/version/:id', async () => ({ api: 'getVersion' }));
+    baseRouter.notFound(async () => ({ error: 'NotFound' }));
+
+    const todoRouter = new Router();
+    todoRouter.get('/', async () => ({ api: 'listTodos' }));
+    todoRouter.post('/create', async () => ({ api: 'createTodo' }));
+    todoRouter.get('/:id', async () => ({ api: 'getTodo' }));
+
+    const taskRouter = new Router();
+    taskRouter.get('/', async () => ({ api: 'listTasks' }));
+    taskRouter.post('/create', async () => ({ api: 'createTask' }));
+    taskRouter.get('/:taskId', async () => ({ api: 'getTask' }));
+
+    const app = new Router();
+    app.includeRouter(baseRouter);
+    app.includeRouter(todoRouter, { prefix: '/todos' });
+    app.includeRouter(taskRouter, { prefix: '/todos/:id/tasks' });
```

Comment:
I feel like this test has a lot of redundant routes that only add noise, we don't want to re test the routing - I think having two routers with one route each and a middleware would would cover the logic we added.